src/Main/Intro.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Email from '../assets/email-icon.png'
+import Gitter from '../assets/gitter-icon.png'
+
+const Intro = (props) => {
+  return (
+    <div className="intro">
+      <h2>A team of blockchain engineers committed to scaling Ethereum.</h2>
+      <p>We are implementing the first <a className="pink" href="https://github.com/ethereum/wiki/wiki/Sharding-FAQ">sharding</a> client for the go-ethereum project.</p>
+      <p>Currently, blockchains do not scale. With Ethereum receiving over <i>a million transactions per day</i>, the network is desperate for a solution that will help sustain its massive growth and adoption.</p> 
```

Comment:
Instead of italics, can we bold a million transactions per day or instead just change the color of it? the italics look a bit weird